docs/faq/connect_scalar_parameters_to_ode.ipynb

```diff
@@ -0,0 +1,238 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "active-ipynb",
+     "remove-input",
+     "remove-output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    import openmdao.api as om\n",
+    "    import dymos as dm\n",
+    "except ImportError:\n",
+    "    !python -m pip install openmdao[notebooks]\n",
+    "    !python -m pip install dymos\n",
+    "    import openmdao.api as om\n",
+    "    import dymos as dm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How do I connect a scalar input to the ODE?\n",
+    "\n",
+    "By default, we recommend that users treat all ODE input variables as if they are _potentially_ dynamic.\n",
+    "This allows the user to use the input as either a dynamic control, or as a static design or input parameter.\n",
+    "By default, parameters will \"fan\" the value out to all nodes.\n",
+    "This allows the partials to be defined in a consistent fashion (generally a diagonal matrix for a scalar input and output) regardless of whether the input is static or dynamic.\n",
+    "\n",
+    "**But** there are some cases in which the user may know that a variable will never have the potential to change throughout the trajectory.\n",
+    "In these cases, we can reduce a bit of the data transfer OpenMDAO needs to perform by defining the input as a scalar in the ODE, rather than sizing it based on the number of nodes.\n",
+    "\n",
+    "## The Brachistochrone with a static input.\n",
+    "\n",
+    "The local gravity `g` in the brachistochrone problem makes a good candidate for a static input parameter.\n",
+    "The brachistochrone generally won't be in an environment where the local acceleration of gravity is varying by any significant amount."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the slightly modified brachistochrone example below, we add a new option to the BrachistochroneODE `static_gravity` that allows us to decide whether gravity is a vectorized input or a scalar input to the ODE."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
Should hide input and output of this block.

Although, I wonder if we should just include the code verbatim instead of calling om.display_source.